src/OpenTelemetry.AutoInstrumentation.Native/util.cpp

```diff
@@ -102,6 +103,30 @@ std::vector<WSTRING> GetEnvironmentValues(const WSTRING& name)
     return GetEnvironmentValues(name, L',');
 }
 
+std::vector<WSTRING> GetEnabledEnvironmentValues(const bool enabled_by_default, const std::unordered_map<WSTRING, WSTRING> values_map)
+{
+    std::vector<WSTRING> values;
+ 
+    for (auto value : values_map)
+    {
+        bool enabled = enabled_by_default;
+        const auto env_value = GetEnvironmentValue(value.second);
+        if (env_value == WStr("true"))
+        {
+            enabled = true;
+        }
+        else if (env_value == WStr("false"))
```

Comment:
I think that everywhere else we are checking also for `0`

Reply:
#2089